pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -86,6 +87,15 @@ mod pallet {
         #[pallet::constant]
         type PalletId: Get<PalletId>;
 
+        /// XCM transfer
+        type XcmTransfer: XcmTransfer<Self::AccountId, Balance, CurrencyId>;
+
+        /// Base xcm transaction weight
+        type BaseXcmWeight: Get<Weight>;
+
+        /// Account manages the staking assets.
+        type Agent: Get<Self::AccountId>;
```

Comment:
we should make it clear that this agent should be on relaychain, also we should prefer `MultiLocation`